resources/guis.yml

```diff
@@ -124,17 +114,6 @@ guis:
   price: Free
   license: GNU GPL
   order: 24
-- name: Cycligent Git Tool
-  url: https://www.cycligent.com/git-tool
```

Comment:
still exists. would vouch to keep

Reply:
Alright, but have you seen the commit description? Maybe take another look - if you still vouch to keep it I'll keep it.

> Website seems to be falling apart. Need to give them your email to get a license (haven't tried doing that), no changelog in sight, Support link is 404, youtube videos at the bottom of the page are private, and their social media is inactive since 2017. Clearly dead.

As a side note, do you think it would be better to comment out instead of delete the ones we agree should go?